src/components/NewObject.vue

```diff
@@ -84,6 +84,11 @@ export default {
         })
         .catch(error => {
           console.log(error);
+          console.log(error.response.data.detail);
+          if (error.response.data.detail) {
+            this.errors = [{ field: "details", message: error.response.data.detail }];
+            return;
+          }
           this.errors = error.response.data.detail
             .filter(detail => detail.loc[1] !== "type")
             .map(detail => {
```

Comment:
Why are you using `error.response.data.detail` is it does not exist after the if condition?

Reply:
Good catch! I actually wanted to check whether it was an array or a string. But I think we need to overhaul the way errors are reported because right now it's extermely confusing